### PR TITLE
master-lps-102472

### PR DESCRIPTION
**Pull request by liuxingye**
https://issues.liferay.com/browse/LPS-102472

### CODE REVIEW

**Comment by liuxingye** [comment]
ci:test:sf

**Comment by austinchiang** [comment]
**Reviewing...** :hourglass_flowing_sand: :sunglasses: :headphones: <br><br>Sent from GitHub :octocat:

**Comment by austinchiang** [comment]
**Pull request submitted:** https://github.com/brianchandotcom/liferay-portal/pull/82449<br></br> <br><br>Sent from GitHub :octocat:

File: modules/apps/depot/depot-test/src/testFunctional/tests/DepotWCIntegration.testcase [hunk 1]
```diff
@@ -112,6 +112,90 @@ definition {
...
+		DisplayPageTemplatesAdmin.addDisplayPage(displayPageName = "Display Page Name");
+
+		PageEditor.addContentDisplay(fragmentName = "Content");
+
+		PageEditor.editContentDisplay(
+			depotName = "Test Depot Name",
+			webcontent = "true",
+			webContentContent = "WC WebContent Depot",
+			webContentTitle = "WC WebContent Title");
+	}
```
Comment (austinchiang) [inline modules/apps/depot/depot-test/src/testFunctional/tests/DepotWCIntegration.testcase]: @liuxingye we should also publish this and view the page template with the web content. Can you create a test improvement for it?

Comment (liuxingye) [reply modules/apps/depot/depot-test/src/testFunctional/tests/DepotWCIntegration.testcase]: Sure, I'll do it here ([LRQA-54647](https://issues.liferay.com/browse/LRQA-54647)) once the new test is merged to upstream.

**Pull request closed without merging**